doc/coq.rst

```diff
@@ -266,6 +271,8 @@ The supported Coq language versions (not the version of Coq) are:
 - ``0.7``: ``(mode )`` is automatically detected from the configuration of Coq
   and ``(mode native)`` is deprecated. The ``dev`` profile also no longer
   disables native compilation.
+- ``0.8``: Support for composition of installed theories; support for vos
```

Comment:
```suggestion
- ``0.8``: Support for composition of installed theories; support for VoS
```

Reply:
I'll change this to `.vos` as monospace.